client/app/initialize.coffee

```diff
@@ -11,27 +11,44 @@ Logging
 
 Send client side errors to server.
 ###
-window.onerror = (msg, url, line, col, error) ->
-    console.error msg, url, line, col, error, error?.stack
-    exception = error?.toString() or msg
-    if exception isnt window.lastError
+initReporting = ->
+    env     = require('imports').env
+    console = console or window.console or {}
+
+    proto  = console
```

Comment:
But if you remove this line and change `proto` by `console` below, it works the same. No?